VTEX - Orders API.json

```diff
@@ -2213,7 +2330,7 @@
                                             "complement": "*** ** *",
                                             "reference": null
```

Comment:
No local onde esse campo é definido, precisa adicionar `"nullable": true` para resolver esse erro do Spectral (`"reference" property type must be string`)

Reply:
Alterado.